frontend/packages/rhoas-plugin/src/components/managed-services-kafka/ManagedKafkas.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import NamespacedPage, {
+  NamespacedPageVariants,
+} from '@console/dev-console/src/components/NamespacedPage';
+import { history } from '@console/internal/components/utils';
+import './ManagedKafkas.css';
+import StreamsInstancePage from '../streams-list/StreamsInstancePage';
+import { ManagedKafkaRequestModel } from '../../models/rhoas';
+import { useActiveNamespace } from '@console/shared';
+import { useK8sWatchResource } from '@console/internal/components/utils/k8s-watch-hook';
+import { ManagedKafkaRequestCRName } from '../../const';
+import {
+  createManagedKafkaConnection,
+  createManagedKafkaRequestIfNeeded,
+  listOfCurrentKafkaConnectionsById
+} from './resourceCreators';
+import { referenceForModel } from '@console/internal/module/k8s';
+import { LoadingBox } from '@console/internal/components/utils';
+
+import { KafkaRequest } from "./types"
+
+const ManagedKafkas = () => {
+  const [currentNamespace] = useActiveNamespace();
+  const [selectedKafka, setSelectedKafka] = React.useState<number>();
+  const [currentKafkaConnections, setCurrentKafkaConnections] = React.useState([]);
+
+  const createKafkaRequestFlow = async () => {
+    await createManagedKafkaRequestIfNeeded(currentNamespace);
+
+    const currentKafka = await listOfCurrentKafkaConnectionsById(currentNamespace)
+    if (currentKafka) {
+      setCurrentKafkaConnections(currentKafka);
+    }
+  }
+
+  React.useEffect(() => {
+    createKafkaRequestFlow()
+  }, []);
+
+  const [watchedKafkaRequest] = useK8sWatchResource<KafkaRequest>({
```

Comment:
we don't need loaded , error state here along with `watchedKafkaRequest`?

Reply:
this will be coming soon when we add error handling